packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/aspnet/challengeintegrationsteps.md

```diff
@@ -1,13 +1,15 @@
 ### 1 - 4: Sign-in and Select Authenticator
-The challenge flow follows the same first four steps as the [enrollment flow](/docs/guides/authenticators-google-authenticator/aspnet/main/#integrate-sdk-for-authenticator-enrollment):
+
+The challenge flow follows the same first four steps as the [enrollment flow](#integrate-sdk-for-authenticator-enrollment):
 
 * Build a sign-in page on the client
 * Authenticate the user credentials
 * Handle the response from the sign-in flow
 * Display a list of possible authenticator factors
 
 ### 5: Check Authenticator Status
-When the user selects the Google Authenticator factor and clicks **Submit**, the form posts back to the `SelectAuthenticatorAsync` method. This checks whether the user is in Challenge Flow or Enrollment Flow.
+
+When the user selects Google Authenticator to authenticate themselves and clicks **Submit**, the form posts back to the `SelectAuthenticatorAsync` method. This checks whether the user is in Challenge Flow or Enrollment Flow.
```

Comment:
"This checks whether the user is in Challenge Flow or Enrollment Flow." - consider rewriting the sentence as it's somewhat fragmented.

Reply:
Have rewritten this a bit.